package.json

```diff
@@ -51,6 +55,7 @@
     "@playwright/test": "^1.40.1",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
+    "@spotlightjs/sidecar": "^1.1.1",
```

Comment:
```suggestion
```
This dep shouldn't be required, the sidecar binary is also part of spotlight package.